raindrops.json

```diff
@@ -0,0 +1,106 @@
+{
+   "#": [
+     "Language implementations vary when one attempts a prime factor on a number with",
+     "and underscore in it.",
+     "In this set of test cases, we have one of these, and the solution would need to",
+     "handle the language implementation."
```

Comment:
We should not have underscores; that's a rubyism that is used purely for readability, and should not leak into the common test inputs/outputs.